python/pyspark/ml/torch/tests/test_distributor.py

```diff
@@ -349,6 +434,13 @@ def test_get_num_tasks_distributed(self) -> None:
 
         self.spark.sparkContext._conf.set("spark.task.resource.gpu.amount", "1")
 
+    # def test_end_to_end_run_distributedly(self) -> None:
```

Comment:
Clean up this?

Reply:
Sorry, that shouldn't have been commented out. Will quickly update adding that test back in